src/entry.rs

```diff
@@ -220,24 +220,12 @@ impl Resolver {
 
     #[must_use]
     pub fn get_dependency_from_entry(&self) -> (Vec<PathBuf>, Vec<PathBuf>) {
-        let mut miss_dependency = vec![];
-        let mut file_dependency = vec![];
-        for entry in &self.entries {
-            let reader = entry.as_ref().stat.read().unwrap();
-            let kind = reader.as_ref().map(|reader| &reader.kind);
-            if let Some(kind) = kind {
-                if kind.is_file() || kind.is_dir() {
-                    file_dependency.push(entry.path.clone());
-                } else {
-                    miss_dependency.push(entry.path.clone());
-                }
-            }
-        }
-        (file_dependency, miss_dependency)
+        todo!("get_dependency_from_entry")
```

Comment:
This method is broken, and it's not used in rspack right now, so I removed it for less confusion.